lexers/testdata/igor.expected

```diff
@@ -0,0 +1,246 @@
+[
+  {
+    "type": "NameDecorator",
+    "value": "#pragma"
+  },
+  {
+    "type": "Text",
+    "value": " rtGlobals=3\n"
+  },
+  {
+    "type": "Error",
```

Comment:
The error tokens indicate that the lexer isn't fully matching.

Reply:
thanks. The line endings are not matching.